src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5475,6 +5475,83 @@
                 "path": "https://creativecommons.org/licenses/by/4.0/"
             }],
             "sources": []
+        },
+        {
+            "profile": "tabular-data-resource",
+            "name": "assn_eia_epacems",
+            "path": "data/assn_eia_epacems.csv",
+            "title": "assn_eia_epacems",
+            "encoding": "utf-8",
+            "mediatype": "text/csv",
+            "format": "csv",
+            "dialect": {
+                "delimiter": ",",
+                "header": true,
+                "quoteChar": "\"",
+                "doubleQuote": true,
+                "lineTerminator": "\r\n",
+                "skipInitialSpace": true,
+                "caseSensitiveHeader": false
+            },
+            "schema": {
+                "fields": [{
+                        "name": "plant_id_eia",
+                        "type": "integer",
+                        "format": "default",
+                        "description": "The unique six-digit facility identification number, also called an ORISPL, assigned by the Energy Information Administration."
+                    },
```

Comment:
It looks like the `plant_id_eia` values in the CSV are being stored as floats (at least some of the time) rather than integers, so this is causing the datapackage validation to fail. You might need to run the dtype fixing function from helpers, and/or add a new section to the datatype dictionary in `pudl.constants` to cover the columns in this new table.

Reply:
fixed!